pipelines/LATISS/ApPipe.yaml

```diff
@@ -0,0 +1,15 @@
+description: AP pipeline specialized for LATISS
+instrument: lsst.obs.lsst.Latiss
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
+    include:
+      - isr
+      - characterizeImage
+      - calibrate
+      - retrieveTemplate
+      - subtractImages
+      - detectAndMeasure
+      - transformDiaSrcCat
+      - diaPipe
```

Comment:
I'd recommend not enumerating these tasks -- they are subject to change, and it will surprise developers if only this pipeline breaks but the others (which only `exclude`) do not.